frontend/apps/reader/modules/readergesture.lua

```diff
@@ -746,7 +746,9 @@ function ReaderGesture:gestureAction(action, ges)
     elseif action == "latest_bookmark" then
         self.ui:handleEvent(Event:new("GoToLatestBookmark"))
     elseif action == "follow_nearest_link" then
-        self.ui:handleEvent(Event:new("GoToPageLink", ges, false, G_reader_settings:isTrue("footnote_link_in_popup")))
+        self.ui:handleEvent(Event:new("GoToPageLink", ges,
+            G_reader_settings:nilOrTrue("swipe_ignore_external_links"),
```

Comment:
I think defaulting to ignoring is an unintuitive default.
```suggestion
           G_reader_settings:isTrue("swipe_ignore_external_links"),
```

Reply:
I disagree for swipe, which is there for a few years and evoke the physical gesture of turning pages to jump to footnotes and back. Having it suddently throw a popup is not a good thing.
(People like you who would still want that can now activate it - but no one else but you complained it was missing :)

Reply:
Well, from the multiswipe gesture, it could be another default or setting. But I don't feel like adding yet another setting for that. So let's have it the same as in readerlink.

Reply:
It breaks the concept of links as well as the internal logic for ignoring external links.

For the gesture manager you could just make two actions: the current one that follows all links, and a new one called `follow_nearest_internal_link`.

Reply:
Ok, went with 2 actions, hardcoded false and hardcoded true, not using the single swipe ignore setting.